README.md

```diff
@@ -1,46 +1,12 @@
-# RG_ActionPatches
```

Comment:
could you remove this file from your PR? I don't want it to overwrite the main one